src/neo/Cryptography/Helper.cs

```diff
@@ -13,56 +13,44 @@ namespace Neo.Cryptography
 {
     public static class Helper
     {
-        internal static byte[] AES256Decrypt(this byte[] block, byte[] key)
+        internal static byte[] AESDecryptNoPadding(this byte[] data, byte[] key, bool ecb, byte[] iv = null)
         {
-            using (Aes aes = Aes.Create())
+            if (!ecb)
             {
-                aes.Key = key;
-                aes.Mode = CipherMode.ECB;
-                aes.Padding = PaddingMode.None;
-                using (ICryptoTransform decryptor = aes.CreateDecryptor())
-                {
-                    return decryptor.TransformFinalBlock(block, 0, block.Length);
-                }
+                if (data == null || key == null || iv == null) throw new ArgumentNullException();
+                if (data.Length % 16 != 0 || key.Length != 32 || iv.Length != 16) throw new ArgumentNullException();
             }
-        }
-
-        internal static byte[] AES256Encrypt(this byte[] block, byte[] key)
-        {
             using (Aes aes = Aes.Create())
             {
                 aes.Key = key;
-                aes.Mode = CipherMode.ECB;
                 aes.Padding = PaddingMode.None;
-                using (ICryptoTransform encryptor = aes.CreateEncryptor())
+                if (ecb)
+                    aes.Mode = CipherMode.ECB;
+                else
+                    aes.IV = iv;
+                using (ICryptoTransform decryptor = aes.CreateDecryptor())
                 {
-                    return encryptor.TransformFinalBlock(block, 0, block.Length);
+                    return decryptor.TransformFinalBlock(data, 0, data.Length);
                 }
             }
         }
 
-        internal static byte[] AesDecrypt(this byte[] data, byte[] key, byte[] iv)
+        internal static byte[] AESEncryptNoPadding(this byte[] data, byte[] key, bool ecb, byte[] iv = null)
```

Comment:
We can use the enum type `CipherMode` instead of the `bool`

Reply:
Done, @shargon, can you check?

Reply:
Is this place I inserted good? In the Helper itself, right?